### PR TITLE
Add sig-spec and deprecate sig-cert

### PR DESCRIPTION
**Pull request by evan2645**
I have prepared this PR for the upcoming SIG-Spec meeting. There, we will:
* Settle on a recurring date/time/frequency
* Identity a SIG lead
* Deliberate over Goals/Non-Goals
* Update this PR with the above information

The first meeting of SIG-Spec is scheduled for October 4th, 2pm Pacific time.

### CODE REVIEW

File: community/sig-spec/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# SIG Spec
+
+A Special Interest Group (SIG) concerning the official SPIFFE specifications.
+
+### Meetings:
+* Date/Time: TBD
+* [Meetings Notes](https://docs.google.com/document/d/1p2BbRWN_7z6nkBMj-h1mAJAJxxKqNeFiV4IplZ_wU4c)
+* Calendar ICS (TBD) (See this [Google support page](https://support.google.com/calendar/answer/37100?co=GENIE.Platform%3DDesktop&hl=en) to learn more about importing .ics files)
```
Comment (evan2645) [inline community/sig-spec/README.md:8]: @suniljames Are we still creating calendars for each SIG, or have we moved to invitations? I couldn't find a calendar for SIG-Spec

File: community/sig-components/README.md [hunk 2]
```diff
@@ -1,4 +1,6 @@
-# SIG Components
+# DEPRECATED - See [SIG-Spec](../sig-spec/README.md)
```
Comment (mattmoyer) [inline community/sig-components/README.md]: This should be SIG Cert Format?

File: community/sig-spec/README.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+### Contact:
+* [Slack Channel](https://spiffe.slack.com/messages/sig-spec/)
+* [Google Group](https://groups.google.com/a/spiffe.io/d/forum/sig-specification)
+
+### Goals:
+* Preside over all [SPIFFE specifications](https://github.com/spiffe/spiffe#spiffe-standards)
+	* Ensure that Pull Requests and Issues are addressed
+	* Make recommendations for changes and updates
+	* Ensure wide interoperability
```
Comment (mattmoyer) [inline community/sig-spec/README.md:18]: Note: this includes owning creation and maintenance of conformance-style test suites.

File: community/sig-components/README.md (hunk 2 above)
Comment (evan2645) [reply community/sig-components/README.md]: Wow good catch, thank you!

File: community/sig-spec/README.md (hunk 3 above)
Comment (evan2645) [reply community/sig-spec/README.md:18]: Thank you, fixed

**Review by y2bishop2y — APPROVED** [review]
LGTM

**Comment by mlakewood** [comment]
+1

**Pull request merged**